src/main/resources/stsjorbsmodResources/localization/zhs/JorbsMod-Character-Strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "stsjorbsmod:CullCharacter": {
+    "NAMES": [
+      "尖塔游魂",
+      "尖塔游魂"
+    ],
+    "TEXT": [
+      "一个恶毒的游魂。 NL 它被传送到这里享用尖塔上的居民。",
+      "NL 你 ~噬咬~ 心脏。",
+      "Navigating an unlit street, you come across several hooded figures in the midst of some dark ritual. As you approach, they turn to you in eerie unison. The tallest among them bares fanged teeth and extends a long, pale hand towards you. NL ~\"dig~ DIG ~dig DIG ~DIG~ DIG~\"~"
```

Comment:
This line is still in English. Intended?

Reply:
I was not quite sure what it means at that time, so I left it untranslated as a reminder. And then forget it... It will be fixed in the new commit.